### PR TITLE
[PDMO-74]: Standardize edit/delete/cancel Buttons

### PR DESCRIPTION
**Pull request by brayoh**
## JIRA TICKET NAME:
[[PDMO-74]: Standardize edit/delete/cancel Buttons](https://issues.openmrs.org/browse/PDMO-74)

## SUMMARY:
Fix the consistency of the delete, save, cancel buttons in terms or color, size and positioning.

### CODE REVIEW

File: app/css/obsadmin.css [hunk 1]
```diff
@@ -489,16 +489,18 @@ li#location.selected a{
   float: none;
   box-shadow: none;
 }
+.full-size{
+  width: 100%;
```
Comment (Farhanisto) [inline app/css/obsadmin.css]: Keep one space between classes

Comment (brayoh) [reply app/css/obsadmin.css]: Thanks for the heads up I somehow missed that, fixing it right away.

File: app/js/components/subAddress/addressForm.jsx [hunk 2]
```diff
@@ -288,26 +288,30 @@ export default class AddressForm extends React.Component {
...
+                        </div>
+                        <div className="col-sm-4">
+                          <button type="button" name="update" onClick={(e) => this.update(e, this.state.uuid)}
+                            className="btn btn-success form-control">Update</button>
+                        </div>
                       </div>
                     </div>
                     : ''
                   }
                   {this.state.action === "new" ?
```
Comment (tolupatrick004) [inline app/js/components/subAddress/addressForm.jsx]: You seem to be using the action state in multiple places. Destructuring the state can make for more clean and readable code. e.g. `const { action, uuid, ...otherInterestingStates } = this.state`

Comment (tolupatrick004) [inline app/js/components/subAddress/addressForm.jsx]: You seem to be using the action state in multiple places. Destructuring the state can make for more clean and readable code. e.g. `const { action, uuid, ...otherInterestingStates } = this.state`

Comment (tduyile04) [inline app/js/components/subAddress/addressForm.jsx:304]: You seem to be using the action state in multiple places. Destructuring the state can make for more clean and readable code. e.g. `const { action, uuid, ...otherInterestingStates } = this.state`

Comment (brayoh) [reply app/js/components/subAddress/addressForm.jsx:304]: I just changed the bootstrap classes, changing the variables assignments means I have to change the entire project to match all across which must be covered in a new ticket.

Comment (brayoh) [reply app/js/components/subAddress/addressForm.jsx:304]: The overall codebase is not so clean, that's why I was suggesting for a refactor sprint.

Comment (tduyile04) [reply app/js/components/subAddress/addressForm.jsx:304]: Got you 👍

**Comment by dkayiwa** [comment]
@brayoh can you look into these merge conflicts?

**Comment by brayoh** [comment]
@dkayiwa I've fixed the conflicts.

**Pull request merged**